packages/object/src/index.ts

```diff
@@ -77,7 +79,7 @@ export class DRPObject implements DRPObjectBase {
 				admins: new Map([[options.peerId, options.publicCredential as DRPPublicCredential]]),
 				permissionless: true,
 			});
-		this.acl = new Proxy(objAcl, this.proxyDRPHandler(DrpType.ACL));
+		this.acl = new Proxy(objAcl, this.proxyDRPHandler<ACL>(DrpType.ACL)) as ProxyHandler<ACL>;
```

Comment:
```suggestion
		this.acl = new Proxy(objAcl, this.proxyDRPHandler<ACL>(DrpType.ACL)) as ACL;
```

why should we change this, and we don't need `acl as ObjectACL` anymore